shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -182,6 +183,81 @@
   return canvases;
 }
 
+void FlutterPlatformViewsController::CompositeWithParams(
+    int view_id,
+    const flutter::EmbeddedViewParams& params) {
+  UIView* touch_interceptor = touch_interceptors_[view_id].get();
+
+  CGRect frame = CGRectMake(0, 0, params.sizePoints.width(), params.sizePoints.height());
+  touch_interceptor.frame = frame;
+  UIView* lastView = touch_interceptor;
```

Comment:
It wasn't immediately clear to me what's `lastView`, the logic here is not completely trivial, maybe we can tell the story better by being a little more verbose on comment and names, maybe something like(pseudo code):

```c++
// Build a chain of UIViews that applies the mutations described by mutatorsStack.
//
// Clips are applied by adding a super view with the CALayer mask. Transforms are applied to the view that's at the
// head of the chain. For example the following mutators stack [T_1, C_2, T_3, T_4, C_5, T_6] where T denotes a
// transform and C denotes a clip, will result in the following UIView tree:
//
// C_2 -> C_5 -> PLATFORM_VIEW
// (PLATFORM_VIEW is a subview of C_5 which is a subview of C_2)
//
// T_1 is applied to C_2, T_3 and T_4 are applied to C_5, and T_6 is applied to PLATFORM_VIEW.
//
// UIView instances used for clip mutations are re-used when possible. [ADD AN EXAMPLE HERE]
//
// Returns the UIView that's at the top of the generated chain(C_2 for the example above).
UIView* FlutterPlatformViewsController::ApplyMutators(ViewMutatorsStack& mutators_stack, UIView* embedded_view) {
UIView* head = embedded_view; // [instead of the current variable named lastView]
...
}
```

Reply:
Done